pkg/virt-launcher/virtwrap/live-migration-source.go

```diff
@@ -1011,8 +987,7 @@ func shouldImmediatelyFailMigration(vmi *v1.VirtualMachineInstance) bool {
 	return shouldFail
 }
 
-func (l *LibvirtDomainManager) asyncMigrate(vmi *v1.VirtualMachineInstance, options *cmdclient.MigrationOptions) error {
-
+func (l *LibvirtDomainManager) migrate(vmi *v1.VirtualMachineInstance, options *cmdclient.MigrationOptions) {
```

Comment:
I think getting rid of `go func()` and moving it(the `go` keyword) to caller could be more clear & idiomatic.